apps/pie-docs/src/_11ty/shortcodes/resourceTable.js

```diff
@@ -44,6 +44,10 @@ const statusSettings = {
         bgColor: statusColours.done,
         status: 'N/A',
     },
+    [statusTypes.TBC]: {
+        bgColor: statusColours.done,
```

Comment:
@FayeCarter Semantically I'm not sure "done" makes sense for deprecated, removed, TBC, etc, and its colour is red? To me, I would expect "done" to mean completed which sounds pretty green to me, so I think we just need a different name for this colour.

@RLumleyJET I just checked Figma and TBC doesn't seem to be on there. Which colour would make most sense to you?

Reply:
This might just be a poor choice of naming on my side.

Reply:
TBC is a new one that we added on Friday. It came up in a discussion that some of the components we've listed may be more web focused than app focused. There will be an audit coming up where we will decide what could be done for these components on the apps side but for now we're using 'tbc' as the status.
Hopefully that was an accurate summary 😬

Reply:
@xander-marjoram I had a catchup with Rachel, we've decided to move tbc and n/a out of 'done' to 'other'